modules/oe_translation_block_field/tests/src/Kernel/BlockFieldTest.php

```diff
@@ -104,8 +104,36 @@ public function testBlockFieldTranslation(): void {
     // Check that the translation was saved correctly on the entity.
     $this->testNode = $this->container->get('entity_type.manager')->getStorage('node')->load($this->testNode->id());
     $translation = $this->testNode->getTranslation('de');
-    $field_block_field = $translation->get('field_block_field')->settings;
-    $this->assertEquals($field_block_field['label'], 'Hello DE');
+    $settings = $translation->get('field_block_field')->settings;
+    $plugin_id = $translation->get('field_block_field')->plugin_id;
+    $this->assertEquals('system_powered_by_block', $plugin_id);
+    $this->assertEquals($settings['label'], 'Hello DE');
```

Comment:
First parameter should be expected and second the actual. I assume here 'Hello DE' is the expected.